scripts/conversion/convert_diffusers_unet.py

```diff
@@ -21,7 +21,7 @@ def setup_converter(args: Args) -> ModelConverter:
     source: nn.Module = UNet2DConditionModel.from_pretrained(  # type: ignore
         pretrained_model_name_or_path=args.source_path,
         subfolder="unet",
-        low_cpu_mem_usage=True,
```

Comment:
Did it cause a problem to have `low_cpu_mem_usage=True` ?

Reply:
Oh haha I think the default for the script was False to avoid accelerate messages. The main reason I had it set to true was when running my computer ran out of ram and killed this process. So for the sake of debugging I converted the weights by setting this to True and then deleting it in that commit